EESSI-install-software.sh

```diff
@@ -336,7 +336,15 @@ else
         if [ -f ${easystack_file} ]; then
             echo_green "Feeding easystack file ${easystack_file} to EasyBuild..."
 
-            ${EB} --easystack ${easystack_file} --robot
+            if [[ ${easystack_file} == *"/rebuilds/"* ]]; then
+                # the removal script should have removed the original directory and created a new and empty one
+                # to work around permission issues:
+                # https://github.com/EESSI/software-layer/issues/556
+                echo_yellow "This is a rebuild, so using --try-amend=keeppreviousinstall=True to reuse the already created directory"
+                ${EB} --easystack ${easystack_file} --robot --try-amend=keeppreviousinstall=True
```

Comment:
@bedroge Pre-creating an empty installation directory and telling EasyBuild to reuse it rather than removing it and creating one itself helps to avoid the permission issues?

Reply:
Ah, more info in #871